contrib/test.sh

```diff
@@ -34,9 +33,10 @@ if [ -n "$BITCOINVERSION" ]; then
     cd integration_test
     ./run.sh
     exit 0
+else
+  # Regular build/unit test.
```

Comment:
Are we sure this has to be either integration or normal test and not both at the same time?

Reply:
I think that was an oversight so far. Integration tests are run for a lot of `bitcoind` versions and unit tests do not depend on that so that these would be just run multiple times for rust stable.

Reply:
Yes I also think it's cleaner to run them separately personally. And it makes it easier to understand potential issues on the CI GUI.